app/src/main/java/woowacourse/movie/domain/policy/TimeDiscountPolicy.kt

```diff
@@ -0,0 +1,19 @@
+package woowacourse.movie.domain.policy
+
+import java.time.LocalDateTime
+
+class TimeDiscountPolicy : DiscountPolicy {
+    override fun discountPrice(price: Int): Int {
+        return price - DISCOUNT_PRICE
+    }
+
+    override fun checkPolicy(date: LocalDateTime): Boolean {
+        if (date.hour !in DAYTIME) return true
+        return false
```

Comment:
```suggestion
        return date.hour !in DAYTIME
```
return 문을 잘 활용하시면 코드를 간소화할 수 있어요